README.md

```diff
@@ -388,19 +388,19 @@ Some standouts:
 | CPU        | yes          | yes           | yes          | yes          | yes            | yes                 |
 | NVIDIA GPU | yes          | yes           | no           | n/a          | no             | experimental        |
 | Google TPU | yes          | n/a           | n/a          | n/a          | n/a            | n/a                 |
-| AMD GPU    | experimental | no            | no           | n/a          | no             | no                  |
-| Apple GPU  | n/a          | no            | experimental | experimental | n/a            | n/a                 |
+| AMD GPU    | experimental | no            | experimental | n/a          | no             | no                  |
```

Comment:
Ah, I think I see the confusion. Here the issue is that "AMD GPU" was meant to mean "ROCM" and "Apple GPU" was meant to mean "Metal". I agree, this is probably clearer.